modules/working-file-repository-service-impl/src/main/java/org/opencastproject/workingfilerepository/impl/WorkingFileRepositoryImpl.java

```diff
@@ -180,7 +180,7 @@ public void activate(ComponentContext cc) throws IOException {
     // Determine which collections should be garbage collected
     List<String> collectionsToCleanUp = null;
     String[] cleanupCollections = StringUtils.split(StringUtils.trimToNull(
-            cc.getBundleContext().getProperty(WORKING_FILE_REPOSITORY_CLEANUP_COLLECTIONS_KEY)));
+            cc.getBundleContext().getProperty(WORKING_FILE_REPOSITORY_CLEANUP_COLLECTIONS_KEY)), ',');
```

Comment:
I did this recently in #3126 like this:

```
String noErrorStateServiceTypesStr = StringUtils.trimToNull((String) properties.get(
            NO_ERROR_STATE_SERVICE_TYPES_CONFIG_KEY));
    if (noErrorStateServiceTypesStr != null) {
      noErrorStateServiceTypes = Arrays.asList(noErrorStateServiceTypesStr.split("\\s*,\\s*"));
```

The advantage of this is that you can also use white spaces after the commas without affecting the results. It also uses the built-in String split functionality - though in this case you have to check for null before splitting, and handle that case accordingly.

Reply:
@KatrinIhler Yes, this is a better solution.